specification/authorization/resource-manager/readme.csharp.md

```diff
@@ -0,0 +1,60 @@
+# C# Authorization
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for DNS.
+
+These settings apply only when `--csharp` is specified on the command line.
+Please also specify `--csharp-sdks-folder=<path to "SDKs" directory of your azure-sdk-for-net clone>`.
+
+## Common C# Settings
+
+``` yaml !$(MultiApi)
+azure-arm: true
+license-header: MICROSOFT_MIT_NO_VERSION
+clear-output-folder: true
+namespace: Microsoft.Azure.Management.Authorization
+output-folder: $(csharp-sdks-folder)/Authorization/Management.Authorization/Generated
+```
+
+## Batch settings
+These settings are for batch mode only: (ie, add `--MultiApi` to the command line )
+
+``` yaml $(MultiApi)
+namespace: Microsoft.Azure.Management.Authorization.$(ApiVersionName)
+output-folder: $(csharp-sdks-folder)/$(ApiVersionName)/Generated
+
+batch:
+- tag: package-2015-07-AzStk
+  ApiVersionName: Api2015_07_01
+- tag: package-2017-10-AzStk
+  ApiVersionName: Api2017_10_01
+```
+
+## Tag: Packages for Azure Stack
```

Comment:
@shahabhijeet why is that? All tags should be in the main Readme, this file is not for tag definition but for C# specifics definition

Reply:
@lmazuel this is the first time we are trying to do something for AzureStack, and we ourselves do not know if this is the right thing to do, as we need feedback from AzureStack team.
So once we have an agreement and things are set in place, these tags will be moved to main readme.md, hence right now they are only C# AzureStack specific and are not ready to be consumed by anyone else outside C# SDK

Reply:
As long as you tell me that it's temporary and experimental, and will not spread across all the Readme, I'm fine :)